_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Kubeflow 1.9: [Topic]"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.9-release/
+author: "Kubeflow 1.9 Release Team, Stefano Fioravanzo"
+---
+
+Kubeflow 1.9 significantly simplifies the development, tuning and management of secure machine learning models and LLMs. Highlights include:
```

Comment:
We might want to add a "short" intro sentence, something like... The Kubeflow project, which is a CNCF Kubernetes-native open source community project for efficient MLOps, is excited to announce the availability of its 17th major software release, Kubeflow 1.9.